content/en/blog/2023/announcing-major-release.md

```diff
@@ -0,0 +1,93 @@
+
+## Highlights
+
+There are some remarkable new features and updates in the first major release. This blog post walks through the major updates in this release.
+
+### Strengthen signing and verification functionalities
+
+From the software producer's perspective, signing a software artifact is the way to ensure authenticity and trust when distributing a software to consumers. Notary Project provides the follow core capabilities for the signing experience.
+
+- Sign artifacts using signing keys stored securely in key management system (KMS) using vendor or customer specific plugins
+- Store signatures using [OCI Image Manifest](https://github.com/opencontainers/image-spec/blob/v1.1.0-rc3/spec.md) for standardization and portability
+- Support two signature envelope formats - [JWS](https://github.com/notaryproject/notaryproject/blob/v1.0.0/specs/signature-envelope-jws.md) and [COSE](https://github.com/notaryproject/notaryproject/blob/v1.0.0/specs/signature-envelope-cose.md) for interoperability
+
+From the software consumer's perspective, verifying the signature of a signed artifact ensure the integrity and authenticity. Notary Project provides the following core capabilities for verification experience:
+
+- Verify signatures using [trust store and trust policy](https://github.com/notaryproject/specifications/blob/v1.0.0/specs/trust-store-trust-policy.md) with fine-tuned configuration for repository specific, and multiple signature verification levels and multiple signature verification levels.  [notation policy](https://notaryproject.dev/docs/cli-reference/notation_policy/) command is also introduced in v1 release to simplify the policy management
+
+### Enhanced debug and troubleshooting experience
+
+[notation inspect](https://notaryproject.dev/docs/cli-reference/notation_inspect/) command is available to get detailed information of signatures associated with the signed artifact in a human readable view.
+
+In addition, `--verbose` flag with all CLI commands, providing troubleshooting capability for developers and users respectively.
+
+### Experimental features
+
+Experimental features are intended for testing and evaluation purposes only and should not be used in production environments. Users can enable experimental features in Notation CLI by setting the environment variable `NOTATION_EXPERIMENTAL` to 1. 
+
+There are two major features which are marked as experimental stage but might be useful in some specific scenarios.
+ 
+- Support [signing, listing and verification with OCI image layout](https://notaryproject.dev/docs/how-to/oci-image-layout/) for signing images before they are pushed to a registry. It enables users to sign and verify images in an air-gapped environment.
+- Support [OCI distribution referrers API](https://github.com/opencontainers/distribution-spec/blob/v1.1.0-rc2/spec.md#enabling-the-referrers-api), which allows Notation CLI to fetch a list of referrers in an efficient and clean manner.
+
+### New specifications
```

Comment:
Suggest move specification to the first item and remove `New` avoid thinking of "Old".